### ISSUE TITLE
Error displayed everywhere on backend after first install

### ISSUE THREAD

**Issue by yoancutillas** [easydigitaldownloads/easy-digital-downloads#5179]
Simply install and activate EDD 2.6.11.

And you will get this error:

> Warning: sprintf(): Too few arguments in C:\...\www\wp-content\plugins\easy-digital-downloads\includes\admin\tracking.php on line 270
> with two buttons: Accept or Deny

If you click on one of those the error won't be displayed anymore. I guess it is a one-time message that is not displayed correctly.
I tested it twice, the bug is 100% reproductible on Wordpress 4.6.1.

**Comment by cklosowski** [comment]
@sunnyratilal Can you try and replicate today? I suspect that it's due to `20%%` in the `sprintf`. I don't think this is new to `2.6.11` though. This section hasn't been changed in a while.

**Comment by sunnyratilal** [comment]
Unable to replicate here

**Comment by sunnyratilal** [comment]
It may just be that the wrap the string in double quotes instead of single quotes?

**Comment by sunnyratilal** [comment]
@yoancutillas which version of PHP are you using? I'm on PHP 7 and can't replicate at all.

My previous comment should be disregarded as it is as escaped anyway.

**Comment by yoancutillas** [comment]
It happened with PHP 5.6.16 on wamp local server and wih PHP 7.0 on my OVH
shared server. I don't know why you can't see the same error.

Yoan Cutillas

yoan.cutillas@gmail.com

2016-11-11 23:10 GMT+01:00 Sunny Ratilal <notifications@github.com>:

**Comment by cklosowski** [comment]
`WAMP` might be the cause, there might be something in the gettext libraries in the Windows version of PHP that is causing that. I'll see if I can kick up a WAMP server this week and get it investigated further.

**Comment by chriscct7** [comment]
It's not in WAMP, as I use it every day

**Comment by chriscct7** [comment]
Or rather its not limited/related to

**Comment by chriscct7** [comment]
Oh I know what it is, sec

**Comment by chriscct7** [comment]
So this isn't a bug in EDD necessarily it's a bug in the translation. I assume @yoancutillas is running the French translation based on other GitHub tickets he's opened.

The line in question reads:
```
sprintf( 'Allow Easy Digital Downloads to track plugin usage? Opt-in to tracking and our newsletter and immediately be emailed a 20%% discount to the EDD shop, valid towards the <a href="%s" target="_blank">purchase of extensions</a>. No sensitive data is tracked.', $extension_url )
```

So notice the `%%`. This is required to escape the `%` so it can be used in sprintf() without being considered a subsitution. 

However, notice that translation is done before the sprintf. 

In the french translation, and probably others, it should be noted that they've translated it as follows:
```
English:
"Allow Easy Digital Downloads to track plugin usage? Opt-in to tracking and our newsletter and immediately be emailed a 20%% discount to the EDD shop, valid towards the <a href=\"%s\" target=\"_blank\">purchase of extensions<\/a>. No sensitive data is tracked."

French:
["Autoriser Easy Digital Downloads \u00e0 suivre anonymement l\u2019utilisation de l\u2019extension ? Acceptez et abonnez-vous \u00e0 notre newsletter et recevez imm\u00e9diatement par e-mail un coupon de remise de 20% pour tout achat dans notre boutique <a href=\"%s\" target=\"_blank\">d\u2019extensions<\/a>. Aucune donn\u00e9e sensible n\u2019est suivie."],
```

This means there is now only 1 `%` there and it is no longer escaped, leading to the error @yoancutillas is seeing.

**Comment by chriscct7** [comment]
The only thing I can think of here, is changing the `%%` to a `%`s, and then having it substitute `%%` for the string as a strintf arg

**Comment by sunnyratilal** [comment]
@chriscct7 PR for ready for a quick test.

**Comment by sunnyratilal** [comment]
@michaelbeil has reviewed. Waiting on unit tests and mergine.

**Comment by sunnyratilal** [comment]
Merged into `master`
